src/resources/CaseAssistConfigs/CaseAssistConfigInterfaces.ts

```diff
@@ -0,0 +1,41 @@
+export enum DocumentSuggestionsStrategies {
+    ITD = 'ITD',
+}
+
+export enum TypingAidsStrategies {
+    StaticValues = 'StaticValues',
+    ValuesFromIndex = 'ValuesFromIndex',
+}
+
+export interface CaseAssistConfigListOptions {
+    page?: number;
+    pageSize?: number;
+}
+
+interface ITDConfigurationModel {
```

Comment:
export all interfaces and types and this file and we're good to go!

Reply:
Done! Thanks for your help. How soon can we make a release for me to use on the admin-ui side?

Reply:
As soon as you merge it will automatically bump the right version according to your commit messages and release the new version in npm. The admin-ui will pick it up automatically in its next build as long as its not a new major version.

Reply:
Very cool. Any special guidelines to follow for said commit message?

Reply:
https://github.com/coveo/platform-client#commit-messages

Reply:
Because we generate changelogs, its cleaner to have one commit per logical change, that we don't have redundancies in the change logs. Don't hesitate to rewrite your branch history with amend, rebase and squash as needed 👍

Reply:
I'm getting errors from the merge build as I've skipped the subject part of the commits when using the cli commit script. Given it let me skip it, I assumed it would be okay. The script should be amended, it shouldn't let me do something that'll bite me in the butt since its sole purpose is to guide the user toward compliant commit messages.

Reply:
The culprit is indeed the missing colon

Reply:
** shamefully lowers pitchfork **